### DISCUSSION TITLE
Where can I find my 'scheme'?

### DISCUSSION THREAD

**Discussion by AIPB07** [supabase/supabase#3309] [Questions]
Hi. Currently trying to set up supabase with flutter following the tutorial. At the section describing how to implement deep linking, i'm asked to provide YOUR_SCHEME and YOUR_HOST.

i.e.
`        
<data          
    android:scheme="io.supabase.flutterquickstart"         
    android:host="login-callback" />`

Can anyone tell me the meaning of these two values and where I can find them for my own project?

Thanks!

**Comment by dshukertjr** [comment]
Hi @AIPB07!

According to [this article](https://www.appsflyer.com/resources/guides/deep-linking-for-mobile-marketers/)
>A URI Scheme is a way to directly open the app if it is installed on the device — this is the same as being able to send mail to a very specific Box if you have a PO Box number.

You can choose any values for `scheme` and `host`. The only thing to look out for here is that any app in the world can claim any scheme+host combination, so make sure yours is as unique as possible and make sure those values match the `Additional URLs` you have set in Supabase console!
